Pipfile

```diff
@@ -22,6 +22,8 @@ flask-talisman = "==1.0"
 gevent = "==21.12.0"
 gevent-websocket = "==0.10.1"
 taipy-config = {git = "https://git@github.com/avaiga/taipy-config.git@develop"}
+kthread = "==0.2.3"
+werkzeug = "==2.0.3"
```

Comment:
Why doo weed to fix werkzeug version ?

Reply:
in 2.1, they remove 'werkzeug.server.shutdown' so the thread couldnt shut down, which cause the sudden break in everything when stopping flask dev server. This does not affect the application when using gevent.